versions/3.1.1.md

```diff
@@ -3792,6 +3829,41 @@ security:
     - read:pets
 ```
 
+###### Security Requirement in a Referenced Document
```

Comment:
Why rely on the implicit anchor generated by Github here?

Reply:
@ralfhandl I thought it was because you did that in one of the suggestions I committed to the another PR!  I hadn't realized we could just do that until I saw _someone_ do it.  Maybe that wasn't you?  The plan is to switch them all over before releasing anyway because the other ones won't work anymore.